jetbrains-rider/build.gradle.kts

```diff
@@ -244,6 +296,8 @@ val resharperDllsDir = tasks.register<Sync>("resharperDllsDir") {
     from(buildReSharperPlugin) {
         include("**/bin/**/$buildConfiguration/**/AWS*.dll")
         include("**/bin/**/$buildConfiguration/**/AWS*.pdb")
+        // TODO: see if there is better way to do this
+        exclude("**/AWSSDK*")
```

Comment:
Why do we exclude this?

Reply:
conflicts with SDK .dlls bundled with rider